CHANGELOG.md

```diff
@@ -118,13 +122,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add support to remote write to Cortex
+- Add support for `Remote Write` to Cortex
 - Added recording rules
 - Add node affinity to prefer not scheduling on master nodes
 - Added `pipeline` tag to _Hearbeat_ alert to be able to see if it affects
   a stable or testing installation at first glance
-- Added initial support for `remote_write` that will eventually be used for
```

Comment:
This is totally just a nitpick, feel free to ignore, but changing entries in released versions feels off, unless to fix something that's plain wrong.

Reply:
We Can keep it. It just thought it was a dupe with the first Line in this release: Add support for `Remote Write` to Cortex